dask/dataframe/tests/test_shuffle.py

```diff
@@ -1538,3 +1539,39 @@ def test_sort_values_timestamp(npartitions):
     result = ddf.sort_values("time")
     expected = df.sort_values("time")
     assert_eq(result, expected)
+
+
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        "Int64",
+        pytest.param(
+            "Float64",
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_120,
+                reason="Support for Float64 columns added in pandas 1.2",
+            ),
+        ),
+        pytest.param(
+            "int64[pyarrow]",
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="Support for ArrowDtypes added in pandas 1.5"
+            ),
+        ),
+        pytest.param(
+            "float64[pyarrow]",
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="Support for ArrowDtypes added in pandas 1.5"
+            ),
+        ),
+    ],
+)
+def test_sort_values_nullable_column(dtype):
+    df = pd.DataFrame({"a": [2, 3, 1, 2, None, None]})
+    df["a"] = df["a"].astype(dtype)
+    # note the partition size means we will have one partition with only pd.NA
```

Comment:
It'd be good to add a check here that there is actually a partition which is all `pd.NA`. It looks like there is in this case, but since this is a key aspect of this test, it'd be good to make sure someone doesn't come along and change partitioning without realizing.